flatfs.go

```diff
@@ -116,8 +116,11 @@ func (fs *Datastore) ShardStr() string {
 }
 
 func (fs *Datastore) encode(key datastore.Key) (dir, file string) {
+	fmt.Printf("key: %s\n", key.String())
```

Comment:
These look like debugging print statements.

Reply:
Yep they are, until we figured out how to fix the issue.